homeassistant/components/matter/select.py

```diff
@@ -527,4 +527,29 @@ def _update_from_device(self) -> None:
         vendor_id=(4447,),
         product_id=(8194,),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=MatterSelectEntityDescription(
+            key="OccupancySensorBooleanStateConfigurationCurrentSensitivityLevel",
+            entity_category=EntityCategory.CONFIG,
+            translation_key="sensitivity_level",
+            options=["low", "standard", "high"],
+            device_to_ha={
+                0: "low",
```

Comment:
Please check the discovery entry before this which is also for vendor 4447 but another product id. So re the esposed details really just "low, mind and high"?

We decided back then in discussion that we want to have it as user friendly as possible and this might need a more detailed mapping

Reply:
Can you please post screenshots on how the details look in the relevant vendor apps?

Reply:
Ok, so we do not know with what they come up but already include it here?